runtime-sdk/modules/evm/src/evm_backend.rs

```diff
@@ -117,7 +117,9 @@ impl<'c, C: oasis_runtime_sdk::Context> EVMBackend for Backend<'c, C> {
         let state = ctx.runtime_state();
 
         let store = storage::PrefixStore::new(state, &crate::MODULE_NAME);
-        let codes = storage::TypedStore::new(storage::PrefixStore::new(store, &state::CODES));
+        let codes = storage::TypedStore::new(storage::HashedStore::<_, blake3::Hasher>::new(
```

Comment:
I don't think this is needed here as the key is always an address which is already hashed?

Reply:
Fixed, thanks!